imaginairy/modules/diffusion/ddpm.py

```diff
@@ -714,14 +714,17 @@ def configure_optimizers(self):
 
 
 def _TileModeConv2DConvForward(
-    self, input: torch.Tensor, weight: torch.Tensor, bias: torch.Tensor  # noqa
+    self,
+    input_tensor: torch.Tensor,
```

Comment:
this has to stay named input since the function its replacing is named that

Reply:
changed it so "noqa" is on the same line as "input"